mp2i/utils/automod.py

```diff
@@ -0,0 +1,128 @@
+from typing import Dict
+
+import logging
+from pathlib import Path
+
+import discord
+import numpy as np
+import onnxruntime as ort
+from tokenizers import Tokenizer
+
+from mp2i import MODEL_DIR
+from mp2i.wrappers.guild import GuildWrapper
+
+logger = logging.getLogger(__name__)
+
+
+def is_toxic(msg: discord.Message) -> bool:
+    """
+    Check if a message is toxic or not.
+    """
+    return _classifier.predict(msg.content, treshold=0.9)
```

Comment:
```suggestion
    return _classifier.predict(msg.content, threshold=0.9)
```
A remplacer aussi ligne 114 et 125

Reply:
Si tu rechanges la modification dans le commit d'après ça va pas marcher :joy: 
Faut aussi modifier les lignes 144 et 125